src/libLookup/Lookup.cpp

```diff
@@ -2592,6 +2594,110 @@ bool Lookup::GetOfflineLookupNodes() {
   return true;
 }
 
+bool Lookup::ProcessGetDirectoryBlocksFromSeed(
+    const vector<unsigned char>& message, unsigned int offset,
+    const Peer& from) {
+  uint64_t index_num;
+  uint32_t portNo;
+
+  if (!Messenger::GetLookupGetDirectoryBlocksFromSeed(message, offset, portNo,
+                                                      index_num)) {
+    LOG_GENERAL(WARNING, "Unable to set GetLookupGetDirectory");
```

Comment:
I think you mean unable to **get** GetLookupGetDirectory, but overall I suggest to keep the usual message "Messenger::GetLookupGetDirectoryBlocksFromSeed failed." (and change the other places similarly)